pattern/constraint/thing/RelationConstraint.java

```diff
@@ -133,7 +133,6 @@ public AlphaEquivalence alphaEquals(RelationConstraint that) {
 
         private final TypeVariable roleType;
         private final ThingVariable player;
-        private final Set<Label> resolvedRoleTypes;
```

Comment:
we no longer require `resolvedRoleTypes`, as was discussed by haikal and max, so I've completely removed them (the state here was unused!)